martian/src/metadata.rs

```diff
@@ -100,24 +102,38 @@ impl RustAdapterInfo {
     }
 }
 
-pub fn make_timestamp(datetime: DateTime<Local>) -> String {
-    datetime.format("%Y-%m-%d %H:%M:%S").to_string()
+pub fn make_timestamp(datetime: impl Into<OffsetDateTime>) -> String {
+    _make_timestamp(datetime.into())
+}
+
+fn _make_timestamp(datetime: OffsetDateTime) -> String {
+    // Convert to local time (if necessary)
+    let datetime = datetime.to_offset(UtcOffset::local_offset_at(datetime).unwrap());
+    datetime.format(DATE_FORMAT).unwrap()
 }
 
 pub fn make_timestamp_now() -> String {
-    make_timestamp(Local::now())
+    make_timestamp(SystemTime::now())
 }
 
 impl Metadata {
-    pub fn new(args: Vec<String>) -> Metadata {
+    pub fn new(mut args: Vec<String>) -> Metadata {
         // # Take options from command line.
         // shell_cmd, stagecode_path, metadata_path, files_path, run_file = argv
+        args.truncate(5);
```

Comment:
this intentionally ignores any additional arguments, and they are silently ignored instead of hitting the new assert below?

Reply:
Truncating was to retain previous behavior.  The assert below is to make sure you have _at least_ 5 arguments.

Reply:
(it would fail later on the `pop` - the assert is there to make the error message more clear)